app.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+(function (){
+  angular
+  .module("toDoList", [])
+  .controller("toDoController",
+  toDoControllerFunction);
+
+  function toDoControllerFunction (){
+      var list=this
+      list.task=["Grocery List" , "Take Car To Shop"];
+      list.detail= ["eggs, milk, butter" , "get itemized receipt, look at breaks, rotate tires"];
+
+      list.new_task={};
+      list.create= function () {
+        var list=this
+        list.task=["Grocery List" , "Take Car To Shop"];
```

Comment:
not sure you need to repeat these arrays here if you are creating something new right?